src/Microsoft.OData.Edm/Vocabularies/Annotations/IEdmTermExtensions.cs

```diff
@@ -0,0 +1,300 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="IEdmTermExtensions.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using Microsoft.OData.Edm.Csdl;
+
+namespace Microsoft.OData.Edm.Vocabularies
+{
+    /// <summary>
+    /// Contains extension methods for <see cref="IEdmTerm"/> interfaces.
+    /// </summary>
+    internal static class IEdmTermExtensions
+    {
+        /// <summary>
+        /// Gets the default value expression for the given <see cref="IEdmTerm"/>.
+        /// </summary>
+        /// <param name="term">The given term.</param>
+        /// <returns>Null or the build Edm value expression.</returns>
+        public static IEdmExpression GetDefaultValueExpression(this IEdmTerm term)

```

Comment:
>GetDefaultValueExpression [](http://example.com/codeflow?start=37&length=25)

We seem to call this frequently -- it would be nice if we could cache the default value for a term rather than recompute each time.